e2e/run-e2e-tests.sh

```diff
@@ -336,6 +337,7 @@ EOM
   read -r -d '' POSTROUTING_RULES_LEADER << EOM
 table ip flannel-ipv4 {
 	chain postrtg {
+    comment "chain to manage traffic masquerading by flannel"
```

Comment:
I think it'd look better if it is aligned with the rest of the related config lines of the chain